README.md

```diff
@@ -550,26 +550,26 @@ processing, dependency model generation, observability portal, etc, and docker i
 #### Steps
 1. Clone the repository to GOPATH.
 ```
-$ mkdir -p $GOPATH/src/github.com/cellery-io/
-$ cd $GOPATH/src/github.com/cellery-io/
+$ mkdir -p $GOPATH/src/github.com/wso2-cellery/
```

Comment:
I think we have to clone this to cellery-io, bcz go code is dependent on the clone location. Therefore we have to have this clone location unchanged to wso2-cellery until we fix the code. Therefore please change the build instruction to have the cellery-io.